### ISSUE TITLE
Ryan Burgess: TBD

### ISSUE THREAD

**Issue by supercycle91** [sacjs/sacjs.github.io#109]
> I spoke to Ryan via Twitter in coming over to Sacramento, possibly with some of his colleagues and doing either a tech talk or one of us could run a panel. Would like to start group discussion on what Ryan would be interested in.

Name: Ryan Burgess
Twitter Handle: burgessdryan
GitHub Username: @ryanburgess
Employer: Netflix
Talk Length: TBD
---

Title:
Brief Description:

**Comment by supercycle91** [comment]
@ryanburgess Could you come down 1/23/2018? We host every 4th tuesday of the month so we are open to anything the date listed above as well.

**Comment by ryanburgess** [comment]
That could work! Would you like us to do a panel / live recording of Front End Happy Hour? Or would you prefer to have a couple engineers speak from Netflix?

**Comment by supercycle91** [comment]
I am very interested in a panel but would be interested in hearing some ideas that would align well with you guys.

**Comment by supercycle91** [comment]
@ryanburgess :point_up: :grinning:

**Comment by ryanburgess** [comment]
Great! It sounds like most of the Front End Happy Hour panelists are open to doing a panel discussion. It might be cool to try and live record an episode at sacjs too. We'd be open to topics to talk about too. Anything you think the audience would be really excited to discuss?

**Comment by supercycle91** [comment]
Sounds great. We have a mix of new web devs and experienced devs so topics can be tricky. I think things along the line of topics like: future of javascript, web performance, do's and don't's of javascript, PWAs, native vs web apps, future of web dev, what skills are important when working at large tech companies like netflix, etc. These are all just ideas but open to suggestions. We can surely swing a live recording as we typically record all talks. For FEHH I noticed you usually moderate, do you want to moderate this panel or would you prefer one of us?

**Comment by ryanburgess** [comment]
Thos are all great topics! I'll share them with the panelists and we can pick one.

I can moderate the panel as we typically record if that works?

**Comment by dlindahl** [comment]
@ryanburgess just for the sake of completeness, I'm commenting on this thread. You're good to do whatever you want to do. Brian told me that all you need is a table and chairs. If there is anything else you need, please let me know! We'll also be video recording this if thats OK with you all.

**Comment by ryanburgess** [comment]
@dlindahl thanks for following up! We are all set for February 27. We are excited to come speak.

Table and chairs would be great!  That's great you'll record video of it. I'm all for it. One thing that I would like, if possible, can we have a copy of the video or audio? I'd like to be able to publish an episode if we can from the event.

**Comment by dlindahl** [comment]
Yeah, that's not a problem. Are you planning on bringing all of the equipment you normally use to record an episode? I only have lapel mics (2 good quality, 1 ok quality)

**Comment by ryanburgess** [comment]
@dlindahl we can bring equipment then. We will require a table then so we can set up all the equipment for recording. Also, do you mind providing details about the event, what time we should be there? How long should we plan to run the panel for? Anything else we should know?

**Comment by ryanburgess** [comment]
I see we are on at 7:05 and have an hour. That's perfect.

**Comment by dlindahl** [comment]
The start time is a rough estimate and you have about as much time as you need. I went to checkout our new venue the other day and only have a couple things you should know.

The ceilings have lots of duct work protruding from them so the projector will be on a table and projecting over your heads. Not a big deal but just be aware so that no one accidentally blinds themselves like I always seem to do.

The room is enclosed with lots of hard surfaces so its prone to be a bit echo-y. The venue assures me that its not a problem. Others have done video recordings in there and have re-booked so it must not sound bad on the recording. I'm hoping that with the 77+ (❗️ ) attendees, all those human bodies will act as sound dampeners. 

I recommend getting there NLT 6:30pm so we can set up your equipment and do any last minute debugging. We have plenty of tables and chairs so no concern there. I should be there by 5:30pm to setup the room and our recording equipment.

**Comment by ryanburgess** [comment]
Hey! That shouldn't be a problem. It is a live show so it can't be perfect audio quality but hopefully, it will sound decent.

We will plan to be there around 6 or 6:30 pm. We will bring mics with us to record to our computers. Will the other mics pick us up enough for the camera and audience? We can also make sure to speak loud.

**Comment by dlindahl** [comment]
The lavalier mics I have are half-way decent. If I space them out accordingly, it should have no problem picking everyone up. We can also trade data files afterwards if need be. My camera has 4 inputs and we usually only use 2. The venue has a handheld mic that we may use for audience questions, but I'm not sure how compatible it is with my setup.

Other than that, is there anything our attendees show know coming in to this tomorrow? Will you be driving the show along yourselves or are you expecting to work entirely off audience participation?

**Comment by dlindahl** [comment]
@ryanburgess any chance you can hook me up with your copy of the podcast audio? My feed for Jem and Brian were great, but the Augustus/Mari is barely useable, and Stacey/Derek is passable.

**Comment by ryanburgess** [comment]
@dlindahl I'd be happy to share it with you.

Do you want the individual tracks? Or I just released the episode http://frontendhappyhour.com/episodes/bar-tending-to-your-codebase/

**Comment by dlindahl** [comment]
@ryanburgess If I could get the individual tracks, that'd be great!

**Comment by ryanburgess** [comment]
Do you want to email me and I can send you the tracks ryanb@netflix.com
